maistra/bazelrc

```diff
@@ -3,3 +3,9 @@ build --cxxopt -w
 
 build --copt -DOPENSSL_IS_BORINGSSL=0
 build --verbose_failures
+
+# Arch-specific build flags, triggered with --config=$ARCH in bazel build command
+build:x86_64 --define=DUMMY=dummy
+build:s390x --//source/extensions/filters/common/lua:luajit2=1 --copt="-Wno-deprecated-declarations" --linkopt='-fuse-ld=bfd' --nostart_end_lib
+build:ppc --@envoy//source/extensions/filters/common/lua:luajit2=1
```

Comment:
Do we want to enable luajit2 for ppc too? If yes, does it need the other compiler flags as in s390x?
It looks line this line has an extra `@envoy`

Reply:
Updated luajit2 flag for p. We definitely need luajit2 for p (luajit doesn't work on p), but I don't have p hardware to verify this (I'll ask someone from p team to take a look at this).